### PR TITLE
Created Wall guide this time moving it to the right folder. 

### PR DESCRIPTION
**Pull request by develroo**
Same pull request to add a Wall guide tutorial but this time I moved the tutorial to a suitable location and edited the index, and added a header to the file so hopefully it will work as intended. 

Not sure if the path allows subfolders but I hope it does as I have added assets to the tutorial folder to keep the whole thing portable. 

Let me know if I need to fix anything. 

Thanks.

### CODE REVIEW

**Comment by Kruptein** [comment]
You can actually check how the page will look when merged by clicking on the following:
image

and browse to your article.

The images do as you somewhat anticipated not render correctly, recommend you checkout `src/pages/learn/general/ui.mdx` for an example, it uses `<Image` tags that do require a bit of extra manual work (e.g. specifying the aspectratio). I should look into updating the blog dependencies as astro (the main framework used) has some new releases that improve some image handling, but we're stuck with this for now.

**Comment by develroo** [comment]
> You can actually check how the page will look when merged by clicking on the following: image
> 
> and browse to your article.
> 
> The images do as you somewhat anticipated not render correctly, recommend you checkout `src/pages/learn/general/ui.mdx` for an example, it uses `<Image` tags that do require a bit of extra manual work (e.g. specifying the aspectratio). I should look into updating the blog dependencies as astro (the main framework used) has some new releases that improve some image handling, but we're stuck with this for now.

Hmm, wasn't sure how the images should be indexed given it was markdown, it looked ok. Let me investigate what looks wrong.

Thanks.

**Comment by develroo** [comment]
Hmmm not allowing me to permissions to see the netify logs. You can review the changes manually if you want. Pretty obvious what I changed.

**Comment by Kruptein** [comment]
Line 54 has the width not in { }

**Comment by develroo** [comment]
> Line 54 has the width not in { }

Whoops my  bad.

**Comment by Kruptein** [comment]
And line 101 doesn't end the Image tag

**Comment by develroo** [comment]
> And line 101 doesn't end the Image tag

Uggg really not used to this. Sorry. We'll get there eventually.

**Comment by develroo** [comment]
Does the path need an absolute reference like `./assets/image.png`  ?

**Comment by Kruptein** [comment]
There are still multiple issues:

- You're missing the import of the Image element
- You're not importing the images into the file, like the example file I referenced

**Comment by Kruptein** [comment]
That said, I'll first upgrade astro like I said I should. It changes a bunch of things in regards to images. From what I can tell you'll be able to revert your images to the `sdf` syntax.

**Comment by Kruptein** [comment]
ok @develroo if you merge/rebase the changes I pushed to master, you should be able to just use what you were doing before.

i.e. `Alt text`

**Comment by develroo** [comment]
You mean switch back to markdown ?

**Comment by Kruptein** [comment]
That's what 

> i.e. `Alt text`

is yes.

You can also use the <Image> syntax, but that is more complex.

**Comment by develroo** [comment]
> That's what
> 
> > i.e. `Alt text`
> 
> is yes.
> 
> You can also use the `<Image>` syntax, but that is more complex.

Ok let me revert back to the markdown version then. Cheers

**Comment by develroo** [comment]
Finally! Seems to look ok now. Phew.. Let us know what you think?

File: src/doc-structure.ts [hunk 1]
```diff
@@ -139,9 +139,10 @@ export const docStructure: Record<string, DocsNavigation[]> = {
                     title: "Creating your first session",
                     path: "/first-session/",
                 },
+               
                 {
-                    title: "RTS-style lighting",
-                    path: "/rts-lighting/",
+                    title: "Wall Masks Guide",
```
Comment (Kruptein) [inline src/doc-structure.ts:144]: You've removed the rts style doc here

Comment (develroo) [reply src/doc-structure.ts:144]: Whoops must have happened when I reverted. My bad. I'll add it back.

**Comment by develroo** [comment]
:crossed_fingers:  Finally a go?

**Pull request merged**